skt/executable.py

```diff
@@ -1004,27 +1004,3 @@ def check_args(parser, args):
                 'the stdio reporter was selected but arguments for the mail '
                 'reporter were provided'
             )
-
-
-def main():
```

Comment:
Where is this being used? Is it being used? Looks like a dead code to me.

Reply:
https://github.com/CKI-project/skt/blob/master/setup.cfg#L33